perfmetrics/scripts/testing_on_gke/examples/dlio/dlio_workload.py

```diff
@@ -0,0 +1,118 @@
+"""This file defines a DlioWorkload (a DLIO Unet3d workload) and provides utility for parsing a json
+
+test-config file for a list of them.
+"""
+
+import json
+
+
+def validateDlioWorkload(workload: dict, name: str):
+  """Validates the given json workload object."""
+  if 'dlioWorkload' not in workload:
+    print(f"{name} does not have 'dlioWorkload' key in it.")
+    return False
+
+  if 'bucket' not in workload:
+    print(f"{name} does not have 'bucket' key in it.")
+    return False
+
+  if 'fioWorkload' in workload:
+    print(f"{name} has 'fioWorkload' key in it, which is unexpected.")
+    return False
+
+  dlioWorkload = workload['dlioWorkload']
+  for requiredAttribute, expectedType in {
+      'numFilesTrain': int,
+      'recordLength': int,
+      'batchSizes': list,
+  }.items():
+    if requiredAttribute not in dlioWorkload:
+      print(
+          f'In {name}, dlioWorkload for {name} does not have'
+          f' {requiredAttribute} in it'
+      )
+      return False
+    if not type(dlioWorkload[requiredAttribute]) is expectedType:
+      print(
+          f'In {name}, dlioWorkload[{requiredAttribute}] is of type'
+          f' {type(dlioWorkload[requiredAttribute])}, expected:'
+          f' {expectedType} '
+      )
+      return False
+
+  for batchSize in dlioWorkload['batchSizes']:
```

Comment:
To shorten code and errors, alias 
```suggestion
  batchSizes = dlioWorkload['batchSizes']
  for batchSize in batchSizes:
```